install.sh

```diff
@@ -43,10 +63,12 @@ rm -rf clients/
 unzip clients.zip
 
 echo -e "${COLOR}----------| Downloading TranslationRecorder|----------${NC}"
+
 URL=$(curl 'https://api.github.com/repos/wycliffeassociates/translationrecorder/releases?per_page=1' | jq -r '.[0] | .assets[].browser_download_url')
```

Comment:
Should now download from https://github.com/Bible-Translation-Tools/BTT-Recorder/releases

Reply:
done